Problems/7568_lhm/7568.py

```diff
@@ -0,0 +1,13 @@
+T = int(input())
+ans = []
+for i in range(T):
+    w, h = map(int, input().split()) 
+    ans.append((w,h))
+
+for i in range(T):
+    count = 1
+    for j in range(T):
+        if i != j:
+            if ans[i][0] < ans[j][0] and ans[i][1] < ans[j][1]:
+                count = count +1
+    print(count)
```

Comment:
백준에서는 이렇게 줘도 통과한 것 같은데, 정확한 출력 Format과 다름.
```suggestion
    print(count,end=" ")
print()
```

Reply:
14번에 print()는 왜 하는거에요?

Reply:
저건 그냥 답변 표시하려고 하는거고 end를 붙여야 좀더 정확한 답변양식인듯